wazo_confgend/confgen.py

```diff
@@ -1,11 +1,9 @@
 # -*- coding: utf-8 -*-
 # Copyright 2011-2022 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
-
-from __future__ import unicode_literals
-
 import logging
 import time
+from typing import List
```

Comment:
Yay typing! You might want to add `from __future__ import annotations` just to make them lazy. No point evaluating them if they aren't used at runtime.